libs/application/templates/driving-license/src/lib/template.ts

```diff
@@ -19,30 +19,19 @@ const dataSchema = z.object({
   type: z.array(z.enum(['general', 'trailer', 'bike'])).nonempty(),
   subType: z.array(z.string()).nonempty(),
   approveExternalData: z.boolean().refine((v) => v),
-  address: z.object({
-    home: z.string().nonempty(),
-    postcode: z.string(),
-    city: z.string().nonempty(),
-  }),
-  user: z.object({
-    name: z.string().nonempty(),
-    phoneNumber: z.string().min(7),
-    nationalId: z.string().refine((x) => kennitala.isPerson(x)),
-    email: z.string().email().nonempty(),
-    country: z.string().nonempty(),
-  }),
   healthDeclaration: z.object({
-    usesContactGlasses: z.boolean(),
-    hasEpilepsy: z.boolean(),
-    hasHeartDisease: z.boolean(),
-    hasMentalIllness: z.boolean(),
-    usesMedicalDrugs: z.boolean(),
-    isAlcoholic: z.boolean(),
-    hasDiabetes: z.boolean(),
-    isDisabled: z.boolean(),
-    hasOtherDiseases: z.boolean(),
+    usesContactGlasses: z.enum(['yes', 'no']),
```

Comment:
Would be nice to create a consts file for this

Reply:
agree ... the application needs a cleanup PR (this includes the translation mappings)